t/t0009-dmesg.t

```diff
@@ -117,6 +117,17 @@ test_expect_success 'logged non-ascii printable characters are unmodified' '
 test_expect_success 'dmesg request with empty payload fails with EPROTO(71)' '
 	${RPC} log.dmesg 71 </dev/null
 '
+# Note: TZ=UTC is set by default for sharness
+test_expect_success 'dmesg with TZ=UTC uses "Z" as offset' '
+	flux dmesg \
+	    | grep -E \
+	    "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}\.[0-9]+Z"
+'
+test_expect_success 'dmesg with TZ=America/Los_Angeles uses tz offset' '
+	TZ=America/Los_Angeles flux dmesg \
+	    | grep -E \
+	    "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}\.[0-9]+-[0-9]{2}:[0-9]{2}"
```

Comment:
my memory isn't good on timezone stuff, but if we're fixing American/Los Angeles in the test, wouldn't the offset in the last chunk be known?  `-08:00`?

Reply:
I think it could be different depending on current daylight savings? Anyway, this was probably good enough and I didn't want a randomly failing test.

The test is failing anyway in some builders - I wonder if they don't support this timezeone